app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+
+def copy_file(command: str) -> None:
```

Comment:
The function name 'copy_file' is not fully descriptive. It would be better if the function name could clearly state what the function does. For example, 'copy_file_from_command' would be more descriptive.